### DISCUSSION TITLE
User Guide?

### DISCUSSION THREAD

**Discussion by rpedrica** [outline/outline#3287] [General]
I'm not sure if I'm missing something but have searched both https://github.com/outline/ and https://www.getoutline.com/ and have not found any user guide or similar for Outline (I'm referring to using the Outline editor, templates, etc., not installation and config). Have I missed it? Can you point me to anything?

**Comment by Limezy** [comment]
https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/

**Reply by rpedrica** [reply]
Thanks but this is related to config of the app and not using the app. There is no info on the editor, templates, etc. at this link.

**Reply by tommoor** [reply]
We should create something this year now that nested document sharing is possible we can create it inside of Outline :) If you have any specific questions then happy to answer

**Reply by rpedrica** [reply]
Thanks Tom, 3 questions:

- Subsections in a doc (eg. H1, H2, etc. can be collapsed and expanded - is there a shortcut for this?
- I can't find a method of changing an existing type of subsection eg. H3 -> H1 (/ works for some items but not all, I can also select the entire text but only get the condensed editor pop menu so for eg. I can't change from H4 to H3)
- is there a way to use html anchors as shortcuts in a doc?
